language/tools/move-mv-llvm-compiler/tests/rbpf-tests/entry-point03.move

```diff
@@ -0,0 +1,22 @@
+// input entry-point03.json
+// abort 8192
```

Comment:
Incidentally, I just chose 8192 arbitrarily, which will work fine. But it might be best to choose an apropos symbolic value from enum `vm_status::StatusCode`. This is what I did for overflow/arithmetic errors where we emit the call to `rt_abort` (that one being `vm_status::StatusCode::ARITHMETIC_ERROR`).

Reply:
Yes, will fix this when I add rt_abort to the default case.